test/BTCUtils.test.js

```diff
@@ -2,7 +2,7 @@
 const BN = require('bn.js');
 const constants = require('./constants');
 
-const BTCUtilsDelegate = artifacts.require('BTCUtilsDelegate');
+const BTCUtilsDelegate = artifacts.require('BTCUtilsTest');
```

Comment:
Why did we introduce `*Test` contracts? Can't we use `*Delegate` contracts?

Reply:
https://github.com/sc-forks/solidity-coverage/issues/234

The `*Test` contracts are contracts, the `*Delegate` contracts are libraries. Pure library calls break the coverage tool